### ISSUE TITLE
Add support for vayu

### ISSUE THREAD

**Issue by ojaghiradin** [openandroidinstaller-dev/openandroidinstaller#270]
**Device informations**
- Brand: Xiaomi
- Phone name: Poco X3 Pro
- Device code (if unknown, leave blank): 
- Model (if several models exist): M2102J20SG

**Device images**  
Please provide links to any available images of the device, such as official stock firmware or custom ROMs you would like to install.  
https://wiki.lineageos.org/devices/vayu/ , https://get.pixelexperience.org/vayu
**Do you own the device and would be willing to test the installer?**
- [ yes ]

**Additional context**  
Please provide any additional context or information that may be helpful in adding support for this device.

**Comment by anon1892** [comment]
Same installation process as #222, I will do that today.
You will be able to test it with branch `dev-flash-additional` (some already / not already merged PR are required to support Xiaomi devices, will all be merged with next update iirc)

**Comment by anon1892** [comment]
Ok, no problem! If you want to test, we can provide you a running version of the program (no need to do anything, just click it and it will start)

But before that, to install custom ROM, you will have to unlock your bootloader. This will reset your device. Make sure you have a backup! There are the steps : 

 - Create a Mi account on Xiaomi’s website. Beware that one account is only allowed to unlock one unique device every 30 days.
 - Add a phone number to your Mi account, insert a SIM into your phone.
 - Enable developer options in `Settings` > `About Phone` by repeatedly tapping MIUI Version.
 - Link the device to your Mi account in `Settings` > `Additional settings` > `Developer options` > `Mi Unlock status`.
 - Download the Mi Unlock app with the link bellow (Windows is required to run the app), and follow the instructions provided by the app. It may tell you that you have to wait, usually 7 days. If it does so, please wait the quoted amount of time before continuing to the next step!
 - After device and Mi account are successfully verified, the bootloader should be unlocked.
 - Since the device resets completely, you will need to re-enable USB debugging to continue : `Settings` > `Additional settings` > `Developer options` > `USB debugging`

**Comment by tsterbak** [comment]
Has been released with 0.5.0.

Feel free to report your experience with the new release.

**Comment by ojaghiradin** [comment]
i will be testing it soon but i dont know how to back up all my data 
can any one help me with that ?
i will test it as soon as i backup my data

**Comment by tsterbak** [comment]
> i will be testing it soon but i dont know how to back up all my data
can any one help me with that ?
i will test it as soon as i backup my data

Hey, thank you that you want to test the installer with your device :)

I can only recommend to backup everything in a manual way. Connect your device to a computer and copy what you want to keep. Automatic backup solutions don't work well from what I can tell. In general on Android and especially on custom ROMs. Only use them when you verified they work for your setup.
Long story short, copy everything you can to your PC, everything else will be gone.

**Comment by ojaghiradin** [comment]
thank you
after backing my data up , i tried to test it but it says i need to a12 installed , i currently have a13 installed on my phone
do i have to downgrade ?

**Comment by tsterbak** [comment]
If you are trying an Android 12 based ROM (like LineageOS 19.1), I guess you should be on Android 12. But thats probably a question @anon1892 can answer better.

**Comment by anon1892** [comment]
> do i have to downgrade ?

You have to be on the same Android version you want to install (or last available for phones no more supported by the manufacturer) (unless exceptions)
It's generally written when you download your ROM : use the version developers recommend

Examples :
For [Lineage 19.1](https://wiki.lineageos.org/devices/vayu/install), Android 12 is needed (so you have to downgrade)
For [crDroid](https://forum.xda-developers.com/t/rom-13-0-crdroid-9-official.4516191/) (A13), latest version (Android 13) is good

**Comment by anon1892** [comment]
Yes, you can ! Download [PE (Android 13)](https://get.pixelexperience.org/vayu), so you don't have to downgrade or whatever (they don't specify a specific version you should be on)
Deprecated only signify there is no maintainer (so bugs may not be patched, but officially supported devices almost never have bugs, so no worries ^^)

**Comment by ojaghiradin** [comment]
thank you thank you thank you
i installed pixel experience today
and i swear to god it was the most easy thing that i have ever done
thank you so much
its amazing

**Comment by tsterbak** [comment]
> thank you thank you thank you
i installed pixel experience today
and i swear to god it was the most easy thing that i have ever done
thank you so much
its amazing

Thanks for the great review! :smile: I'm happy it worked so well for you! :)

**Comment by anon1892** [comment]
> thank you thank you thank you

You're welcome, happy to help!

@tsterbak Can you remove `untested` metadata and mark `vayu` as tested in the README ?
